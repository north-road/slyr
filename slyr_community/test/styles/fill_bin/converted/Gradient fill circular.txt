[{'class': 'QgsGradientFillSymbolLayer',
  'color1': '0,0,255,255',
  'color2': '35,35,140,255',
  'color_type': '1',
  'direction': 'cw',
  'reference_point1': '0.5,0.5',
  'reference_point2': '1,0.5',
  'stops': '0.25;0,0,255,255;rgb;ccw',
  'type': '1'}]
