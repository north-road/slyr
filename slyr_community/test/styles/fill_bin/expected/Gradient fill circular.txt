{'levels': [{'angle': 90.0,
             'enabled': True,
             'fill_color': {'B': 255,
                            'G': 0,
                            'R': 0,
                            'dither': False,
                            'is_null': False,
                            'stream_offset': '0x105',
                            'type': 'HsvColor',
                            'version': 1},
             'gradient_type': 'circular',
             'intervals': 5,
             'locked': False,
             'outline': {'color': {'B': 110,
                                   'G': 110,
                                   'R': 110,
                                   'dither': True,
                                   'is_null': False,
                                   'stream_offset': '0x146',
                                   'type': 'RgbColor',
                                   'version': 1},
                         'enabled': True,
                         'line_type': 'solid',
                         'locked': False,
                         'stream_offset': '0x134',
                         'symbol_level': 0,
                         'type': 'SimpleLineSymbol',
                         'version': 1,
                         'width': 1.0},
             'percent': 0.75,
             'ramp': {'algorithm': 0,
                      'color1': {'B': 140,
                                 'G': 35,
                                 'R': 35,
                                 'dither': False,
                                 'is_null': False,
                                 'stream_offset': '0xa7',
                                 'type': 'HsvColor',
                                 'version': 1},
                      'color2': {'B': 255,
                                 'G': 0,
                                 'R': 0,
                                 'dither': False,
                                 'is_null': False,
                                 'stream_offset': '0xd6',
                                 'type': 'HsvColor',
                                 'version': 1},
                      'stream_offset': '0x5f',
                      'type': 'AlgorithmicColorRamp',
                      'version': 1},
             'stream_offset': '0x4d',
             'symbol_level': 0,
             'type': 'GradientFillSymbol',
             'version': 1}],
 'stream_offset': '0x0',
 'symbol_level': 0,
 'type': 'MultiLayerFillSymbol',
 'version': 2}
